ldflags_test.go

```diff
@@ -0,0 +1,25 @@
+package astibundler
```

Comment:
Since the flag logic will be moved to the `main` package, those tests will be moved to the `main` package as well